apps/src/assetManagement/icon_list_generator.rb

```diff
@@ -0,0 +1,69 @@
+require 'yaml'
```

Comment:
When/how is this file run? Might deserve a comment. Also wonder if this better belongs in some sort of bin directory somewhere instead of living next to source code.